src/mame/includes/segas32.h

```diff
@@ -256,6 +261,9 @@ class segas32_state : public device_t
 	uint8_t m_sprite_control_latched[8];
 	uint8_t m_sprite_control[8];
 	std::unique_ptr<uint32_t[]> m_spriteram_32bit;
+	std::unique_ptr<int[]> m_prev_startx;
+	std::unique_ptr<int[]> m_prev_endx;
+	std::unique_ptr<int[]> m_bgcolor_line;
```

Comment:
You need to use explicit-sized types for things that you’re saving for portability.